Tech-Talk-LA/TechTalk
# Make fetch request to get matches

## Description 
We implemented the ability to make a request to get the matches data from the backend. This data then gets stored in the top component's state (via hooks), and passed down into the Feed component to be rendered as a list of FeedItems. 

## What we changed
The data is no longer hard-coded in the front end, but comes from the database. 

# How to test
npm run server (starts the server)
npm start (starts the expo server)

ALSO, if you are working off of an ios device (not sure if this is also true for ios simulators) you will need to replace 'localhost' with your computer's public ip address. This is an ios thing (they don't allow http, and you can't change that from expo). Android devices should be ok.
